tests/system/src/testscriptapp1/java/mymod/myapp1/FXMLScriptTest.java

```diff
@@ -0,0 +1,303 @@
+                                       false,   // boolean secondaryButtonDown,
+                                       false,   // boolean synthesized,
+                                       false,   // boolean popupTrigger,
+                                       false,   // boolean stillSincePress,
+                                       null     // PickResult pickResult
+                                     )
+                      );
+
+        // mainStage.setScene(scene);
+        // mainStage.show();
+        Platform.exit();
+    }
+
+    // show engine invocations with script text and their Bindings
+    static void dumpEvalInformation()
+    {
+        System.err.println("\nListing eval() invocation information (invocationList):");
+
+        Iterator<RgfPseudoScriptEngine> it = RgfPseudoScriptEngine.getEnginesUsed().iterator();
+        while (it.hasNext())
+        {
+            RgfPseudoScriptEngine rpse = it.next();
+            ArrayList invocationList   = rpse.getInvocationList();
+            System.err.println("ScriptEngine: ["+rpse+"]");
+
+            Iterator<InvocationInfos> itEval = invocationList.iterator();
+            int count = 1;
+            while (itEval.hasNext())
+            {
+                System.err.println("\teval() invocation # "+count+": ");
+                InvocationInfos entry = itEval.next();
+                System.err.println(entry.toDebugFormat("\t\t"));    // indentation
+                count++;
+                System.err.println();
+            }
+        }
+    }
+
+    static void assertCorrectInvocations()
+    {
+            // test only creates one engine for a script controller
+        Util.assertTrue( "exactly one pseudo script engine instance", RgfPseudoScriptEngine.getEnginesUsed().size()==1);
+        RgfPseudoScriptEngine rpse = RgfPseudoScriptEngine.getEnginesUsed().get(0);
+
+        ArrayList invocationList   = rpse.getInvocationList();
+        Util.assertTrue( "exactly nine script engine invocations", invocationList.size()==9);
+
+        final String FILENAME  = "javax.script.filename";
+        final String ARGV      = "javax.script.argv";
+        final String EVENT     = "event";
+
+        final String IDBUTTON  = "idButton";
+        final String IDROOT    = "idRoot";
+        final String LOCATION  = "location";    // always FXML File hosting script controller code
+        final String RESOURCES = "resources";   // always null in this test
+
+
+        for (Integer invocation=1; invocation<= invocationList.size(); invocation++ )
+        {
+            InvocationInfos entry= (InvocationInfos) invocationList.get(invocation-1);
+            String                   script=entry.script;
+            TreeMap<Integer,TreeMap> scopes=(TreeMap) entry.bindings;
+
+            TreeMap<String,Object> engineBindings = scopes.get(100);
+            TreeMap<String,Object> globalBindings = scopes.get(200);
```

Comment:
In general, except for variables that are essentially lists of similar things, it's generally preferred not to try to line up the assignment statements. I noticed a few places above, but here it's really noticeable.